angularjs-portal-home/src/main/webapp/my-app/layout/widget/partials/lol.html

```diff
@@ -0,0 +1,35 @@
+<div class='widget-body'>
+   <div class='row'>
+      <div class='col-xs-6 center' ng-repeat='item in config.links'>
```

Comment:
limit 4? 

What should 3 look like? Triangle? What should 2 look like, centered? One?